app/assets/javascripts/add-forms.js

```diff
@@ -0,0 +1,55 @@
+$(document).on('ready page:load', function(){
+
+  addForm('#add-guest', 'guests');
+  addForm('#add-key', 'keys');
+  addForm('#add-place', 'places');
+
+  $('#form-drop-down').on('click', '#cancel-form', function(event){
+    event.preventDefault();
+    console.log($('#form-drop-down'))
+    $('#form-drop-down').html('<hr>');
+  });
+
+})
+
+var addForm = function(selector, type) {
+  $(selector).on('click', function(event){
+
+    $.ajax({
+      method: 'GET',
+      url: '/'+ type +'/new'
+    })
+    .done(function ( response ) {
+      response = response + dropDownFooter;
+      $('#form-drop-down').append(response);
+    })
+    .fail(function(jqXHR,textStatus){
+      /* code ... */
```

Comment:
Maybe console.log or alert or something. Doing nothing is not very user-friendly or debuggable.